addons/website/tests/test_page_manager.py

```diff
@@ -21,6 +21,6 @@ def test_01_page_manager(self):
         self.start_tour(url, 'website_page_manager', login="admin")
         self.start_tour(url, 'website_page_manager_session_forced', login="admin")
 
-        alternate_website = self.env['website'].search([], limit=2)[1]
+        alternate_website = self.env['website'].search([('company_id', '=', self.env.company.id)], limit=2)[1]
```

Comment:
```suggestion
        alternate_website = self.env['website'].search([('name', '=', 'My Website 2')], limit=2)[1]
```
Let's go with this, searching explicitly what we are looking for. Code was not ideal before indeed :grimacing: but let's not go with another "indirect" way (could still break later) of searching the website